data/hai missions.txt

```diff
@@ -1061,3 +1061,252 @@ mission "Hai Honeymoon"
+			`	You do your best in telling her of the historical presence of crime on Earth, and of how little was done to address the root of the problem.`
+			`	"I see..." She says. "I just thought it would be fun to hear about another large planet like Hai-home. Sorry for asking about it, Captain, I'm sure having to speak of it wasn't a pleasant feeling."`
+			`	Although you're sure that even in Hai space there is social inequality, she probably can't fathom that the home planet of a species could be in such bad shape, considering the Hai even go as far as to send aid to the Unfettered on a regular basis.`
+					goto board
+			label board
+			`	You two reach the job board and decide to start looking for jobs suited for Nanachi's ship.`
+			choice
+				`	"So, do you mind if I help out?  I know the job board can be confusing the first few times you check it."`
+				`	"Would you like me to accompany you to the job board?"`
+			`	"Thanks, but I have to do this on my own. I promise I won't mess this up by getting greedy again," She approaches the board on her own, and it becomes clear that it's much easier to get a job here: she's barely started talking to one of the desk workers before receiving a long list of jobs for smaller ships. She is certainly still very tired, but this seems to have cheered her up a bit, it is the first time she can get a job without your help after all.`
+			`	While you thought she would've picked one or two jobs quickly and headed back to her ship, she keeps going through them and double checking the information for each one, as if something about them bothers her.`
+			`	"What's wrong? Are you struggling to pick between two different jobs?" you ask.`
+			`	"These all seem very easy and I could even accept many at a time, but the problem is their pay isn't very good." She grabs some worn papers from her pocket, and shows it to you: they're the documents for her loan. "The bank back home would only give me enough for the ship if I agreed on a short-term loan. I lost two days before you came along to help me, and if I don't get some better paying job soon, I won't even be able to pay the interest."`
+			`	As you're trying to figure out a way to get the most out of the available missions, you feel a tap on your shoulder, and turn to see a Hai standing beside a human couple. The couple introduce themselves as Xavier and Vanessa, who are here with their Hai friend Narancia. "We overheard your conversation and are willing to pay a little extra for a job. We're part of a group organizing a big festival in Allhome to celebrate the friendship between humans and Hai, and we just got a last-minute idea: we've already arranged for lots of human cuisine to be transported there, and we'd like to get our hands on some spices the Quarg have in the ringworld here."`
+			`	They tell Nanachi that they won't be going along, they already have a transport to Allhome, and will stay here a bit longer to try and advertise the event.`
```

Comment:
This ends rather abruptly. No "Great, I'll take the job!" from Nanachi? No "The Quarg are on `<stopovers>`, then you can bring the spices to `<destination>`."? No mention of the deadline?